languages/python/exercises/concept/comments/.meta/design.md

```diff
@@ -0,0 +1,32 @@
+# Design
+
+## Goal
+
+The goal for this exercise is to teach about comments.
+
+## Things to teach
+
+- How to comment in program.
+
+## Things not to teach
+
+
+
+## Concepts
+
+It is good programming habit.
+
+## Prerequisites
+
+concepts of comment is helpful for this exercise.
```

Comment:
How is this a prerequisite? Prerequisites mean other concept exercises that should be completed before this one.